doc/source/extending-jupyterhub.rst

```diff
@@ -81,9 +81,10 @@ configure JupyterHub to build off of this image.
 
    .. code-block:: bash
 
-          numpy=1.12.1
-          scipy=0.19.0
-          matplotlib=2.0
+          pip==9.0.1
```

Comment:
Nope, that makes the image non-reproducible. We should try to use == as much as possible.